### ISSUE TITLE
py27-trial and py35-trial environments failures on Travis and AppVeyor

### ISSUE THREAD

**Issue by The-Compiler** [pytest-dev/pytest#1989]
See e.g. https://travis-ci.org/pytest-dev/pytest/jobs/165320324:

```
____________________ test_trial_testfunction_todo_property _____________________
testdir = <Testdir local('/tmp/pytest-of-travis/pytest-0/testdir/test_trial_testfunction_todo_property0')>
    def test_trial_testfunction_todo_property(testdir):
>       pytest.importorskip('twisted.trial.unittest')
/home/travis/build/pytest-dev/pytest/testing/test_unittest.py:353: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/travis/build/pytest-dev/pytest/.tox/py35-trial/lib/python3.5/site-packages/twisted/trial/unittest.py:12: in <module>
    from twisted.trial._synctest import (
/home/travis/build/pytest-dev/pytest/.tox/py35-trial/lib/python3.5/site-packages/twisted/trial/_synctest.py:23: in <module>
    from twisted.trial import itrial, util
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    """

    from __future__ import division, absolute_import

>   import zope.interface as zi
E   AttributeError: module 'zope' has no attribute 'interface'
```

**Comment by nicoddemus** [comment]
Thanks! I meant to take a look in #1979 but didn't have the time, so opening an issue is better.

**Comment by nicoddemus** [comment]
I investigated this for some time today and here's what I found so far:

1. It doesn't depend on pytest's version.
2. It doesn't happen in python 3.4.
3. It can be reproduced with:

  ```
  $ pytest test_unittest.py::test_trial_testfunction_skip_property test_unittest.py::test_trial_testcase_skip_property
  ```

  This command makes the **second** test to fail. Changing the order of the tests again makes the **second** to fail.

Those tests execute a trial `TestCase` in-process so I think that's the cause although I don't know what is the actual problem.
